hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/ECReplicationConfig.java

```diff
@@ -82,14 +82,17 @@ public ECReplicationConfig(int data, int parity, EcCodec codec,
   }
 
   /**
-   * Create an ECReplicationConfig object from a string representing the
-   * various parameters. Acceptable patterns are like:
+   * Create an {@link ECReplicationConfig} object from a string representing
```

Comment:
I don't think linking to `ECReplicationConfig` from the javadoc of `ECReplicationConfig` (i.e. the same class) adds any value.